code/libs/discord/discord.cpp

```diff
@@ -272,5 +272,15 @@ void shutdown()
 	shutdown_discord();
 }
 
+void set_presence_string(SCP_string text)
```

Comment:
This ought to be `const SCP_string&`, like `set_presence`

Reply:
done